jenkins/check_copyright.sh

```diff
@@ -59,7 +60,7 @@ if [ -e $ERRFILE ]; then # if error file exists then there were errors
     >&2 echo "--- ERRORS ---"
     cat $ERRFILE 1>&2 # send output to stderr so it can be redirected as error if desired
     >&2 echo "--------------"
-    exit 1 # tell the caller there was an error (so Jenkins will fail the CI task)
```

Comment:
If you want to keep make it more future-proof, you can remove the name of the CI system we are using. I don't think it is critical, we can keep Azure here.

Reply:
done

Reply:
I think you might still need `exit 1`, otherwise Azure may not pick the failure up.